test/libsolidity/syntaxTests/inlineAssembly/natspec_multi.sol

```diff
@@ -0,0 +1,19 @@
+function f() pure {
+    /// @unrelated
+
+    /// @before
+    ///
+    /// @solidity a   memory-safe-assembly b    c
+    ///           d
```

Comment:
Oh... I'm just realizing that apparently the natspec parser reads this as a tag "before" with value
``@solidity a   memory-safe-assembly b    c``...
That's not ideal...
But maybe just fine, if I also output the value in the error message.